blueprints/secops/bitwarden-cli/ops2deb.yml

```diff
@@ -0,0 +1,13 @@
+name: bitwarden-cli
+version: 2024.6.1
+homepage: https://bitwarden.com/help/cli/
+summary: Tool to access and manage a bitwarden vault
+description: |-
+  The Bitwarden command-line interface (CLI) is a powerful, fully-featured
+  tool for accessing and managing your bitwarden vault.  Most features that you
+  find in other Bitwarden client applications (desktop, browser extension, etc.)
+  are available from the CLI.
+fetch: https://github.com/bitwarden/clients/releases/download/cli-v{{version}}/bw-linux-{{version}}.zip
+script:
```

Comment:
You can remove the first line of your script here. And for the second one, use `install` here instead of `script`. Look at `blueprints/terminal/yq/ops2deb.yml` for instance.

Reply:
Hm… when stripped the resulting executable does not work any longer, hence I took over this pattern to skip stripping from other examples in the repository.

Anything I am overlooking here?

Reply:
Alright then, you're right, I thought it was a copy/paste mistake. Could you use install for the second line thought!

Reply:
Done.